app/workers/upload_removal_worker.rb

```diff
@@ -0,0 +1,12 @@
+class UploadRemovalWorker
+  include Sidekiq::Worker
+
+  def perform
+    Upload.where(
+      post: nil,
+      comment: nil
+    ).where(
+      ['created_at > ?', 11.hours.ago]
```

Comment:
Probably should be a scope (`orphaned` maybe?) on `Upload`, it's best to avoid directly using `where` outside of the model.